ROBT 3341 - SCARA/scara_control.c

```diff
@@ -101,10 +108,20 @@ void moveJoint(void) {
 #	This function prompts the user for a specific x,y coordinate and left or 
 # right arm configuration.
 #
-# Last Modified: September 30, 2025
+# Last Modified: Dec 02, 2025 - Peter
 # ---------------------------------------------------------------------------*/
 void moveTCP(void) {
+	double joint1, joint2, x, y;
+	char armSol;
+
+	printf("Please enter the joint angles you would like to move to (X,Y,armSolution)"
+		   "(0 or 1 for arm solution, being left or right)");
+	scanf("%lf,%lf, %c", &x, &y, &armSol);
 
+	inverseKinematics(x, y, &joint1, &joint2, armSol);
+	setScaraPosition(joint1*JOINT1_RESOLUTION, joint2*JOINT1_RESOLUTION, m0Speed, m1Speed);

```

Comment:
While technically fine, You'd likely want to separate this like so
```C 
setScaraPosition(
  joint1*JOINT1_RESOLUTION,
  joint2*JOINT2_RESOLUTION,
  m0Speed,
  m1Speed
);
```
Doing this, you see that you have the wrong joint resolution for value 2